ext/storage_sources/azure_store/CMakeLists.txt

```diff
@@ -46,3 +54,7 @@ target_link_libraries(
     PRIVATE
         azure_container_util
 )
+
+if(HAVE_UNITTEST)
```

Comment:
This should already be in the cmake list, why is it here?

Reply:
deleted was somehow still around - Thanks for picking this up! https://github.com/wiredtiger/wiredtiger/pull/8702/commits/e48383f1ca180dc7222bcd3c57585e590dfcd30f